Packs/GroupIB_ThreatIntelligenceAttribution/IncidentFields/incidentfield-GIB_Admiralty_Code.json

```diff
@@ -1,37 +1,81 @@
 {
-    "associatedToAll": false,
-    "associatedTypes": [
-        "GIB Brand Protection Phishing",
-        "GIB Brand Protection Phishing Kit",
-        "GIB Compromised Account",
-        "GIB Compromised Card",
-        "GIB Data Breach",
-        "GIB OSI Git Leak",
-        "GIB OSI Public Leak",
-        "GIB Targeted Malware"
-    ],
-    "caseInsensitive": false,
-    "cliName": "gibadmiraltycode",
-    "closeForm": false,
-    "content": true,
-    "description": "The Admiralty System or NATO System is a method for evaluating collected items of intelligence. It consists of a two-character notation, evaluating the reliability of the source and the assessed level of confidence on the information.",
-    "editForm": true,
-    "group": 0,
-    "hidden": false,
-    "id": "incident_gibadmiraltycode",
-    "isReadOnly": false,
-    "locked": false,
-    "name": "GIB Admiralty Code",
-    "neverSetAsRequired": false,
-    "ownerOnly": false,
-    "required": false,
-    "sla": 0,
-    "system": false,
-    "threshold": 72,
-    "type": "shortText",
-    "unmapped": false,
-    "unsearchable": true,
-    "useAsKpi": false,
-    "version": -1,
-    "fromVersion": "6.0.0"
-}
+	"aliasTo": "",
+	"aliases": null,
+	"associatedToAll": false,
+	"associatedTypes": [
```

Comment:
in order to make sure this change is applied, you will have to modify also the `systemAssociatedTypes` key in each of JSON files of the incident fields where you added new type associations, with the same list of incident types that appear under `associatedTypes`. New fields don't need this modification, only modified fields.

Reply:
Hello @idovandijk,
Thank you for your review.
In each modified field file, do I need to reconcile systemAssociatedTypes and associatedTypes?
For example, if in 
```
“associatedTypes”: [
  GIB OSI Vulnerability”.
],
```

then

```
“systemAssociatedTypes":[
  “GIB OSI Vulnerability”.
],
```

Reply:
Exactly :)

Reply:
Thank you, it's done